.mockery.yaml

```diff
@@ -0,0 +1,11 @@
+with-expecter: true
+disable-version-string: true
+dir: ../mockadmin
```

Comment:
[Question] What is the benefit of the separate folder compared to admin/mockery folder?

For me
`admin/mockery` would be removed and regnerated every time?

Reply:
i would like the last folder to be mockadmin to keep that package name,  so it would be admin/mockadmin.
I put it in a different folder to have the admin folder cleaner but not a strong opinion to have it in admin.

Reply:
If you have strong preference for mockadmin then we can leave it like that. No worries.